android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -54,14 +53,26 @@ fun SurveyQuestionsScreen(
         )
     }
 
+    showExitDialog.let {
+        if (showExitDialog) {
+            ExitAlertDialog(
+                onConfirmClick = {
+                    viewModel.onExitDialogConfirm()
+                    onCloseClick()
```

Comment:
`onCloseClick` event here is duplicated with the one in `SurveyQuestionsScreenContent` but their meanings are different: one goes through the confirmation dialog and another one points to the navigation action. I think we should have them under different names.

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/122/commits/34df858b7cdc74f4a9252b232270a14b2ef99b47